### PR TITLE
added info about with_current_organizations to the person docs

### PR DESCRIPTION
**Pull request by fatemeh-affinity**
Ticket: https://app.asana.com/0/1202786177801211/1203333072548898

with_current_organizations is added as query parameter to the get person/s api to retrieve and return the person's current organizations.

### CODE REVIEW

**Review by mscrivo — APPROVED** [review]
Approving to get your build to work, but I want us to make sure someone is properly maintaining this repo, at the very least, we should:

* Use only latest version of Ruby
* Make sure gems in use are up to date and don't contain security vulns
* Scan upstream repo commits since forking to make sure we're not missing any important security patches.

**Comment by mscrivo** [comment]
If Ruby is just used for building the static html and github's servers are serving it up, then less concerning .. still we should keep it relatively in sync with affinity repo since most devs are only going to have latest versions of Ruby installed and won't have to install another version to work with this repo.

**Comment by fatemeh-affinity** [comment]
> 
We're ruby 3.2 for affinity codebase, we can update the ruby version to the same thing in so people don't be forced to update an older version of ruby for this proj. Then I we should also remove the builds for all ruby2.X
I can take care of it

File: source/includes/_persons.md [hunk 1]
```diff
@@ -124,19 +124,21 @@ Dates of the most recent and upcoming interactions with a person are available i
...
+| organization_ids         | integer[]   | An array of unique identifiers of organizations that the person is associated with.                                                                                                                                                                                                                  |
+| opportunity_ids          | integer[]   | An array of unique identifiers of opportunities that the person is associated with.                                                                                                                                                                                                                  |
+| current_organization_ids | integer[]   | An array of unique identifiers of companies that the person is associated with and are indicated as their current companies.                                                                                                                                                                         |
```
Comment (marcypark) [inline source/includes/_persons.md:137]:
```suggestion
| current_organization_ids | integer[]   | An array of unique identifiers of organizations that the person is currently associated with according to the Affinity Data: Current Organization in-app column.                                                                                                                                                                       |
```

File: source/includes/_persons.md [hunk 2]
```diff
@@ -232,16 +234,17 @@ curl "https://api.affinity.co/persons" \
...
+| with_interaction_persons        | boolean | false    | When true, persons for each interaction will be returned. Used in conjunction with `with_interaction_dates`                                                                                                                                         |
+| with_opportunities              | boolean | false    | When true, opportunity IDs associated with this person will be returned.                                                                                                                                                                            |
+| with_current_organizations      | boolean | false    | When true, the IDs of person's current organizations will be returned.                                                                                                                                                                              |
```
Comment (marcypark) [inline source/includes/_persons.md:243]:
```suggestion
| with_current_organizations      | boolean | false    | When true, the organization IDs of each person's current organizations (according to the Affinity Data: Current Organizations column) will be returned.                                                                                                                                                                              |
```

File: source/includes/_persons.md [hunk 3]
```diff
@@ -232,16 +234,17 @@ curl "https://api.affinity.co/persons" \
...
+| with_interaction_dates          | boolean | false    | When true, interaction dates will be present on the returned resources. Only persons that have interactions will be returned.                                                                                                                       |
+| with_interaction_persons        | boolean | false    | When true, persons for each interaction will be returned. Used in conjunction with `with_interaction_dates`                                                                                                                                         |
+| with_opportunities              | boolean | false    | When true, opportunity IDs associated with this person will be returned.                                                                                                                                                                            |
```
Comment (marcypark) [inline source/includes/_persons.md:242]:
```suggestion
| with_opportunities              | boolean | false    | When true, opportunity IDs will be returned for each person.                                                                                                                                                                            |
```

File: source/includes/_persons.md [hunk 4]
```diff
@@ -299,12 +302,13 @@ Fetches a person with a specified `person_id`.
...
+| Parameter                  | Type    | Required | Description                                                                                                 |
+| -------------------------- | ------- | -------- | ----------------------------------------------------------------------------------------------------------- |
+| person_id                  | integer | true     | The unique ID of the person that needs to be retrieved.                                                     |
+| with_interaction_dates     | boolean | false    | When true, interaction dates will be present on the returned resources.                                     |
+| with_interaction_persons   | boolean | false    | When true, persons for each interaction will be returned. Used in conjunction with `with_interaction_dates` |
+| with_opportunities         | boolean | false    | When true, opportunity IDs associated with this person will be returned.                                    |
+| with_current_organizations | boolean | false    | When true, the IDs of person's current organizations will be returned.                                      |
```
Comment (marcypark) [inline source/includes/_persons.md:311]:
```suggestion
| with_current_organizations | boolean | false    | When true, the IDs of this person's current organizations (according to the Affinity Data: Current Organizations column) will be returned.                                      |
```

**Review by marcypark — COMMENTED** [review]
Let's also
- [ ] add the with_current_organizations parameter to sample requests/responses
- [ ] add the following to source/includes/_changelog.md:

```
## 2023-02-03

- Added the ability to retrieve Current Organization column data on [Persons](#persons).
```

**Pull request merged**